mmv1/templates/terraform/examples/network_services_edge_cache_service_advanced_beta.tf.tmpl

```diff
@@ -0,0 +1,189 @@
+resource "google_storage_bucket" "dest" {
+  provider             = google-beta
```

Comment:
tests are failing because provider = google-beta is tagged here but the test is not labeled as being a beta test

Reply:
see https://github.com/ScottSuarez/magic-modules/blob/2787513e50afad7ec184363da3cc4c5c64fb9eb0/mmv1/api/resource/examples.go#L132

Reply:
While making that change I realised that the field is supported in the GA api already - I have removed references to beta in my PR.